django/db/models/expressions.py

```diff
@@ -1848,6 +1849,16 @@ def get_group_by_cols(self):
         return group_by_cols
 
 
+class Exclusion(Enum):
```

Comment:
`Exclusion` can be misleading, especially as its importable from `django.db.models`. I will rename it to the `WindowFrameExclusion`.